### PR TITLE
Hotfix/analytics

### PR DESCRIPTION
**Pull request by saumyakaran**
## Goals

To fix product tracking (bloated event property state)

Events and user properties:
https://docs.google.com/spreadsheets/d/1SnWK0nI05DI6UR1ITQemkbx2du3Z7w2CwMfs_H26sVI/edit#gid=771066228

### CODE REVIEW

**Comment by prastut** [comment]
FYI this seems to being merged into master over a branch that Sahil is working on that hasn't been merged to master.

**Comment by saumyakaran** [comment]
> FYI this seems to being merged into master over a branch that Sahil is working on that hasn't been merged to master.

Yep, it's done so that render bot deploys this PR. When merging I'll change this to `fixes/calculator-page-sprint1`

**Review by sahilnanda1995 — APPROVED** [review]

File: components/common/layouts/dashboard.js [hunk 1]
```diff
@@ -160,11 +164,13 @@ const withDashboardLayout = (children) => {
 						}
 
 						const setStateAndTrack = (details) => {
-							trackEvent(Events.USER_ACCOUNT_SELECTION, {
-								user: {
-									...details,
-									stashId: address,
-								},
+							setUserProperties({
+								selectedStashId: address,
```
Comment (sahilnanda1995) [inline components/common/layouts/dashboard.js:168]: @saumyakaran 
different key names used, in _base.js_ `stashId` and here `selectedStashId`

Comment (saumyakaran) [reply components/common/layouts/dashboard.js]: Updated https://github.com/buidl-labs/yieldscan-frontend/pull/127/commits/0a21b6e3ac0fb4fac08a0d6b4d0d05724b92de63

**Pull request merged**